client/src/language/diagnosticsSupport.ts

```diff
@@ -13,7 +13,7 @@ import path from 'path'
 import { logger } from '../lib/src/utils/OutputLogger'
 import { commonDirectoriesVariables } from '../lib/src/availableVariables'
 
-const supportedSources = ['Pylance', 'shellcheck']
+const supportedSources = ['Flake8', 'Pylance', 'Pylint', 'shellcheck']
```

Comment:
I somehow agree, however we are not doing so for shellcheck, since we're not "officially" supporting it. Should we follow the same reasoning?

I'm also expecting to spend a lot of time on it, as the integration tests have been giving lot of trouble for diagnostics in the past.

Reply:
I think it's worth having a basic test for each one to maintain our compatibility. This would be the time to officially declare this support :)
I think it's worthy of a separate ticket/PR